docs/_docs/Developers/build.md

```diff
@@ -5,52 +5,84 @@ order: 1
 toc: true
 ---
 
-## Ubuntu 18.04/18.10 and Linux Mint 19
+## Ubuntu 18.04/18.10
 
-* Install the dependencies:
-```
+Fork https://github.com/PhoenicisOrg/phoenicis on github before building.
+
+#### Dependencies:
+- git
+- maven
+- java11+
+- fakeroot
+
+#### Building:
+```bash
+# Update repository
 sudo apt update
+
+# Install dependencies
 sudo apt install git maven openjdk-11-jdk fakeroot
-```
 
-* Optional: if your distribution does not support Java 10+ **or** if you need to create packages, grab the latest JDK 
-```
- wget https://download.java.net/java/GA/jdk11/28/GPL/openjdk-11+28_linux-x64_bin.tar.gz -O /tmp/openjdk-11+28_linux-x64_bin.tar.gz
- sudo tar xfvz /tmp/openjdk-11+28_linux-x64_bin.tar.gz --directory /usr/lib/jvm
- export JAVA_HOME="/usr/lib/jvm/jdk-11"
-```
- 
+# Clone your phoenicis repository
+git clone https://github.com/<YOUR_GITHUB_FORK_REPO>/phoenicis.git
 
-* Clone the repository from GitHub:
-```
-git clone https://github.com/PhoenicisOrg/phoenicis.git
-```
+# Sanity-check your JAVA_HOME variable
+echo JAVA_HOME
+export JAVA_HOME=PATH # if needed, should be set automatically.
```

Comment:
Is this line optional for the user? If yes please comment it out so that a user doesn't accidentally execute if when he doesn't need to. Otherwise I think you should rephrase the comment to make its meaning more clear.

Reply:
Created https://github.com/PhoenicisOrg/phoenicis/pull/1860/commits/110b77df287b2710a5de941eaf7094fc2d07c244

Dunno how to make it more clear, feel free to change it as you see fit.

Reply:
So basically the whole line is optional?
Then you should comment it out, otherwise a user who accidentally copies and pastes the line will accidentally destroy his `JAVA_HOME` right?